python/foglamp/services/core/api/configuration.py

```diff
@@ -42,19 +42,24 @@
             the list of known categories in the configuration database
 
     :Example:
-            curl -X GET http://localhost:8081/foglamp/category
-            curl -X GET http://localhost:8081/foglamp/category?root=true
+            curl -sX GET http://localhost:8081/foglamp/category
+            curl -sX GET http://localhost:8081/foglamp/category?root=true
+            curl -sX GET 'http://localhost:8081/foglamp/category?root=true&children=true'
```

Comment:
@MarkRiddoch  please confirm if this is okay, or we need allchildren=true | fulltree=true

Reply:
I think this is fine